packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -466,6 +466,10 @@ export interface IMergeTreeAttributionOptions {
  * @sealed
  */
 export interface AttributionPolicy {
+	readonly _brand?: unique symbol;
+}
+
+export interface AttributionPolicyInternal extends AttributionPolicy {
```

Comment:
i don't understand the need for the rename here. nothing is the merge tree file is exported in index

Reply:
would this also help clean up the casting?

Reply:
this interface and the options object are exported (would be the options that users pass to Client indirectly through sharedstring via their data store runtime options). moving those interfaces to a separate definition file is probably better (similar to `mergeTreeDeltaCallback.ts` for example). but that's some minor internal organization.

I'm also not a huge fan of the opaque type stuff. But I think the bugpit of introducing compat-breaking tags into typetests for internal changes opening us up to actually making breaking changes without realizing it is more dangerous.

that said I'm open to not making this change. I was just really grossed out by how many forwardCompat tags I needed to add to change the signature of `getAll` in the extensibility PR I'm working on